.clang-format: change SpaceBeforeParens style

fbrausse opened the pull request:
This is still compatible with clang-11.

fbrausse committed:
.clang-format: change SpaceBeforeParens style

lucasccordeiro reviewed (approved):
@fbrausse: are you going to apply it to the entire codebase in this PR to keep consistency?

fbrausse committed:
apply clang-format

Commands used for this modification:

	find src -iname '*.h' -o -iname '*.c' -o -iname '*.cpp' -o -iname '*.hpp' -iname '*.hh' | grep -v src/ansi-c/cpp | grep -v src/clang-c-frontend/headers/ | xargs /usr/lib/llvm/11/bin/clang-format -style=file -i -fallback-style=none
	find unit -iname '*.h' -o -iname '*.c' -o -iname '*.cpp' -o -iname '*.hpp' -iname '*.hh' | xargs /usr/lib/llvm/11/bin/clang-format -style=file -i -fallback-style=none

rafaelsamenezes reviewed (approved):
LGTM! We can ignore codacy warning.

lucasccordeiro commented:
Thanks for submitting this PR, @fbrausse.
